docs/source/user_guide/release_notes.md

```diff
@@ -1,5 +1,28 @@
 # Release note
 
+## v0.8.4rc1
+
+This is the first release candidate of v0.8.4 for vllm-ascend. Please follow the [official doc](https://vllm-ascend.readthedocs.io/en/) to start the journey. From this version, vllm-ascend will follow the newest version of vllm and release every two weeks. For example, if vllm releases v0.8.5 in the next two weeks, vllm-ascend will release v0.8.5rc1 instead of v0.8.4rc2.
```

Comment:
> From this version, vllm-ascend will follow the newest version of vllm and release every two weeks. For example, if vllm releases v0.8.5 in the next two weeks, vllm-ascend will release v0.8.5rc1 instead of v0.8.4rc2.

we should doc some on release policy: https://vllm-ascend.readthedocs.io/en/latest/developer_guide/versioning_policy.html#release-window